nevergrad/common/sphere.py

```diff
@@ -826,6 +855,8 @@ def metric_pack_big_conv(x, budget=default_budget):
     "Riesz_blursum_lowconv_loworder",
     "Riesz_blursum_lowconv_midorder",
     "Riesz_blursum_lowconv_highorder",
+    "max_pooling",
+    "pooling"
```

Comment:
All methods in the list have the same signature, so that they can be used by quasi-randomize.
Please use the same signature.